src/components/CheckboxGroup/readme.md

```diff
@@ -272,3 +272,46 @@ const CheckBoxGroupProfile = () => {
 
 <CheckBoxGroupProfile />
 ```
+
+##### Checkbox Group horizontal
+
+```js
+import React from 'react';
+import { CheckboxGroup } from 'react-rainbow-components';
+
+const options = [
+    { value: 'checkboxOne', label: 'Checkbox One', disabled: false },
+    { value: 'checkboxTwo', label: 'Checkbox Two', disabled: false },
+    { value: 'checkboxThree', label: 'Checkbox Three', disabled: false },
+];
+
+class CheckboxGroupTry extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { values: [] };
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
```

Comment:
for new examples, we are using functions with hooks

Reply:
ok, done with arrow function, not sure how you want to do it, not a lot of references. If you prefer "normal" functions just let me know.